apps/swirl-docs/src/lib/docs/src/MergedYamlOperations.ts

```diff
@@ -39,7 +39,7 @@ function isYamlFile(file: string) {
 function createSpecPath(spec: string) {
```

Comment:
as this function is the same as in multipleDocs i think we can share the functionality here and export one of them and import in the other file when needed

Reply:
removed unused code